src/hooks/useLoadClientScript.ts

```diff
@@ -80,7 +80,7 @@ export function useLoadClientScript(props: NostoScriptProps) {
         document.body.appendChild(script)
       }
     }
-  }, [shopifyMarkets])
+  }, [shopifyMarkets?.marketId, shopifyMarkets?.language])
```

Comment:
add support for re-evaluation with shopifyMarkets reference stays the same, but fields change